templates/template.yaml

```diff
@@ -207,7 +232,6 @@ Resources:
       - LayersStack
       - StateMachineStack
       - StreamProcessorStack
-      - WebUIStack
```

Comment:
should we keep the `WebUIStack` here with an IF for the web UI? I think the goal here is to avoid you receiving the email saying "Hey welcome to the thing, this is your user/temp password" before the UI stack is actually created (so the email having a url that is not accessible yet).